content/tutorials/manage-the-rippled-server/fix-sqlite-tx-db-page-size-issue.md

```diff
@@ -0,0 +1,173 @@
+# Fix SQLite Transaction Database Page Size Issue
+
+`rippled` servers with full transaction history (or a very large amount of transaction history) and a database that was initially created with a `rippled` version earlier than 0.40.0 (released January 2017) may encounter a problem with their SQLite database page size that stops the server from operating properly. Servers that store only recent transaction history (the default configuration) and servers whose database files were created with `rippled` version 0.40.0 and later are not likely to encounter this problem.
+
+This document describes steps to detect and correct this problem if it arises.
+
+## Background
+
+`rippled` servers store a copy of their transaction history in a SQLite database. Before version 0.40.0, `rippled` configured this database to have a capacity of roughly 2TB. For most uses, this is plenty. However, full transaction history back to ledger 32570 (the oldest ledger version available in the production XRP Ledger history) is likely to exceed this exceed the SQLite database capacity. `rippled` servers version 0.40.0 and later create their SQLite database files with a larger capacity, so they are unlikely to encounter this problem.
```

Comment:
I'm going to hold off on that for now, because:

- That content isn't in this branch yet because it's from a PR started after this one. I'd have to rebase onto master.
- I don't think the link adds much info beyond what's already mentioned in the parenthesis, and it might distract from the task at hand.